server/test/gateways/judges.gateway.spec.ts

```diff
@@ -493,148 +501,273 @@ describe('JudgesGateway', () => {
+            wsMain.on(JudgesSocketEvents.EVENTS_SUGGESTION, (data) => {
+              expect(data.status).toBe(ResponseStatus.Unauthorized);
+              resolve();
+            }),
+          );
         });
 
-        await new Promise<void>((resolve) =>
-          wsMain.on('newEvents', (data) => {
-            expect(data.status).toBe(ResponseStatus.BadRequest);
-            resolve();
-          }),
-        );
-      });
+        it('should not suggest events with negative number of points', async () => {
+          wsBlue.emit(JudgesSocketEvents.EVENTS_SUGGESTION, {
+            fightId: fight.id,
+            judgeId: fight.blueJudge.id,
+            events: events,
+            redPlayerPoints: -1,
+            bluePlayerPoints: bluePlayerPoints,
+          });
 
-      it('should add events to started fight and send back fightEndConditionFulfilled', async () => {
-        fight.state = FightState.Running;
+          await new Promise<void>((resolve) =>
+            wsBlue.on(JudgesSocketEvents.EVENTS_SUGGESTION, (data) => {
+              expect(data.status).toBe(ResponseStatus.BadRequest);
+              resolve();
+            }),
+          );
+        });
 
-        wsMain.emit('newEvents', {
-          fightId: fight.id,
-          judgeId: fight.mainJudge.id,
-          events: events,
-          redPlayerPoints: redPlayerPoints, // now will have 6 points > fight.pointsToEnd
-          bluePlayerPoints: bluePlayerPoints,
+        it('should not suggest events to scheduled', async () => {
+          fight.state = FightState.Scheduled;
+
+          wsBlue.emit(JudgesSocketEvents.EVENTS_SUGGESTION, {
+            fightId: fight.id,
+            judgeId: fight.blueJudge.id,
+            events: events,
+            redPlayerPoints: redPlayerPoints,
+            bluePlayerPoints: bluePlayerPoints,
+          });
+
+          await new Promise<void>((resolve) =>
+            wsBlue.on(JudgesSocketEvents.EVENTS_SUGGESTION, (data) => {
+              expect(data.status).toBe(ResponseStatus.BadRequest);
+              resolve();
+            }),
+          );
         });
 
-        for (const ws of [wsMain, wsRed, wsBlue]) {
+        it('should not suggest events to finished fight', async () => {
+          fight.state = FightState.Finished;
+
+          wsBlue.emit(JudgesSocketEvents.EVENTS_SUGGESTION, {
+            fightId: fight.id,
+            judgeId: fight.blueJudge.id,
+            events: events,
+            redPlayerPoints: redPlayerPoints,
+            bluePlayerPoints: bluePlayerPoints,
+          });
+
           await new Promise<void>((resolve) =>
-            ws.on('fightEndConditionFulfilled', (data) => {
+            wsBlue.on(JudgesSocketEvents.EVENTS_SUGGESTION, (data) => {
+              expect(data.status).toBe(ResponseStatus.BadRequest);
+              resolve();
+            }),
+          );
+        });
+
+        it('can suggest events to running fight', async () => {
```

Comment:
To taka drobnostka, ale myślę, że warto trzymać się konwencji z nazywaniem testów przez "should", a nie "can", ponieważ to sugeruje, że kod testu może działać, ale w sumie nie musi ;)